### ISSUE TITLE
ng add store imports the environment wrong

### ISSUE THREAD

**Issue by tdeschryver** [ngrx/platform#1159]
## Minimal reproduction of the bug/regression with instructions:

Create a new application and add the store:
```
ng new splendit-app
cd splendit-app
ng add @ngrx/store
```

This will import the environment from 
```ts
import { environment } from '../../environments/environment';
```
## Expected behavior:

This should be one level up
```ts
import { environment } from '../../../environments/environment';
```

## Other information:

This is the same bug which is fixed earlier inside schematics - #1090 

## I would be willing to submit a PR to fix this issue

[x] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

**Comment by brandonroberts** [comment]
Is this not already fixed in master?

**Comment by tdeschryver** [comment]
The schematics `ng generate store ...` is fixed in master, but the `ng add` implementation to decide the environment path is wrong. 

I'm waiting on a response in #920, because if we don't offer the `module` support I would suggest to hard code the environment path inside `ng add`.

**Comment by brandonroberts** [comment]
I don't see a reason not to provide `--module` support for `ng add`. By default, I think it should just look for the `app.module.ts` though

**Comment by tdeschryver** [comment]
You were right, this was resolved with #1085 and #1088, I missed those... 
Next time I'll check the history, but the good thing is that this is resolved! 😅
